Documentation/Signoffs/BrakePedalSystem.md

```diff
@@ -0,0 +1,38 @@
+- Must have wires able to handle 5V and 20mA to meet the expected voltage range of 0-5V and the expected current of 20mA. 
+- Shall have an accurate measure of the brake pedal position of at least 1% accuracy to ensure the simulation is an accurate representation of real life.
+- Must comply to IEEE 576-13 section on loose wires and splicing of wires.
+- Must have microcontroller with at least four analog or four digital in ports to read both the gas pedal sensors and the brake pedal sensors.
+- Must have microcontroller able to be powered by a 12V source to comply with the power systems design to stay at 12V.
+- Must have microcontroller with UART capability to communicate with the master MCU. 
+- Must have microcontroller able to convert analog to digital in at least 100ms to be consider real time. (Shown in master MCU signoff)
+
+### Wiring Schematic
+
+![BrakePedalSchematic](https://user-images.githubusercontent.com/117474294/217134906-68ba4112-b1f6-4c9f-809d-a28d7ae2ceb4.png)
+
+_Figure 1: The wiring schematic with the desired local MCU and OEM sensors for both pedals being sent to it. 
+
+### Analysis
+The customer wants the brake pedal to be stock so to measure the position I would have to install an external sensor or find and use a sensor that already measures the brake pedals position. The car has a ABS decelerator sensor also know as a brake stroke sensor to compare the position of the brake pedal to the rate at which the wheels are spinning to determine if the ABS system should engage or not[1]. Since this sensor is measureing the position of the brake pedal it would be able to give us this data in analog if we splice into the output wires of the sensor. One brake stroke sensors sends its output in a range 3.35-3.98V and the other has 2.77-1.58V. These ranges are within the constraints and corresponds to the brake pedals position. These ranges are low because the sensor is not able to use its full range of motion. The wires use to splice into the sensors will be 22 AWG which can handle 300V and 7A which is able to handle the 0-5V and 20mA expected. Arduino boards have a step size of 4.88mV on analog inputs meaning the shortest range of 3.35-3.98V has 126 steps. That means the system has an accuracy of 0.79% which meets the constraints. The same wire splicing tools from the gas pedal will be used here to ensure we meet the IEEE standard. The arduino uno has six analog input pins which is enough for the four analog signals from the gas and brake pedals. The arduino uno has to be powered by a 7-12V supply which corresponds to the power systems 12V its suppling everywhere. An arduino uno can talk to other arduino boards using UART. The uno has a processing speed of 16MHz which is 62.5ns for one cycle and it takes 13 cycles to convert analog to digital taking 0.8125 microseconds which is faster then the 100ms constraint. 
+
+### Testing 
```

Comment:
Did you check that a mid press gives a reading in the middle? Are we sure that it is a linear relationship?